src/net/net.zig

```diff
@@ -259,16 +259,7 @@ pub const SocketAddr = union(enum(u8)) {
         }
     }
 
-    /// returns:
-    /// - array: the string, plus some extra bytes at the end
-    /// - integer: length of the string within the array
-    pub fn toString(self: Self) struct { [53]u8, usize } {
-        var buf: [53]u8 = undefined;
-        const len = self.toStringBuf(&buf);
-        return .{ buf, len };
-    }
-
-    pub fn toStringBounded(self: Self) std.BoundedArray(u8, 53) {
+    pub fn toString(self: Self) std.BoundedArray(u8, 53) {
```

Comment:
For my own understanding. What advantage does returning `std.BoundedArray(u8, 53)` has over `struct { [53]u8, usize }`?

Also could it be possible to document why capacity of 53?

Reply:
if you look at the implementation of BoundedArray youll see its actually the same as the struct  { ... } approach we had previously